pkg/config/config.go

```diff
@@ -679,6 +685,23 @@ func load(config Config, origin string, loadSecret bool) error {
 		}
 	}
 
+	// If this variable is set to true, we'll use DefaultPython for the Python version,
+	// ignoring the python_version configuration value.
+	if ForceDefaultPython == "true" {
+		defaultPython, err := strconv.Atoi(DefaultPython)
```

Comment:
I'm not sure it's necessary to check this, since `DefaultPython` is the default value set on the "python_version" config key anyway (see https://github.com/DataDog/datadog-agent/blob/45fea0f26b3b4ead3ccab644d53282ef8f9d34b1/pkg/config/config.go#L144)